multicluster/controllers/multicluster/member/clusterset_controller.go

```diff
@@ -173,32 +231,18 @@ func (r *MemberClusterSetReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&mcv1alpha2.ClusterSet{}).
 		WithEventFilter(generationPredicate).
 		WithOptions(controller.Options{
-			MaxConcurrentReconciles: common.DefaultWorkerCount,
+			MaxConcurrentReconciles: 1,
 		}).
 		Complete(r)
 }
 
-func (r *MemberClusterSetReconciler) createOrUpdateRemoteCommonArea(clusterSet *mcv1alpha2.ClusterSet) error {
-	currentCommonArea := r.remoteCommonArea
+func (r *MemberClusterSetReconciler) createRemoteCommonArea(clusterSet *mcv1alpha2.ClusterSet) error {
 	newLeader := clusterSet.Spec.Leaders[0]
 
 	clusterID := common.ClusterID(newLeader.ClusterID)
 	url := newLeader.Server
 	secretName := newLeader.Secret
 
-	if currentCommonArea != nil {
-		if r.installedLeader.clusterID == newLeader.ClusterID && r.installedLeader.serverUrl == newLeader.Server &&
-			r.installedLeader.secretName == newLeader.Secret {
-			klog.V(2).InfoS("No change for leader cluster configuration")
-			return nil
-		}
-
-		klog.InfoS("ClusterSet update", "old", currentCommonArea.GetClusterID(), "new", newLeader)
-		klog.InfoS("Stopping old RemoteCommonArea", "cluster", clusterID)
-		currentCommonArea.Stop()
```

Comment:
Question - when serverUrl or Secret changes, do we have a way to update them without stopping commonArea? If not, seems we should still keep the code to check if the current commonArea != nil and stop it.

Reply:
No, we passed the secret to commonArea manager and initialize the go client, so I didn't see there is a way to update them without stopping. common area stop for update part is actually moved to the Reconcile() https://github.com/antrea-io/antrea/pull/5351/files#diff-d2f1e28c34f19f40190c020679752f63a69522cc837119c24417ef10e6b528feR142-R145, no need to handle here again.